oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -180,11 +183,58 @@ def _create_relateditem_audio(self, mi: MediaFile) -> MODSv34:
 
         return ri
 
+    def _populate_narrator_image(self):
+        for img in MediaFile.objects.filter(
+            Q(item=self._item) & Q(file_type__file_code="image_submaster")
+        ).order_by("sequence"):
+            self.locations.append(
+                LocationOH(url=img.file_url, label="Image of Narrator")
+            )
+
+    def _populate_interview_content(self):
+        fc_to_label = {
+            "pdf_master": "Interview Full Transcript (PDF)",
+            "text_master_transcript": "Interview Full Transcript",
+            "text_master_biography": "Interviewee Biography",
+            "text_master_interview_history": "Interview History",
+            "pdf_master_appendix": "Appendix to Interview",
+            "text_master_appendix": "Appendix to Interview",
+            "pdf_master_resume": "Narrator's Resume",
+        }
+
+        for f in MediaFile.objects.filter(
+            Q(item=self._item) & Q(file_type__file_code__in=fc_to_label.keys())
```

Comment:
This probably could be simplified to not use `Q`:
`filter(item=self._item, file_type__file_code__in=fc_to_label.keys())`
Assuming output is unchanged and tests pass :)

Reply:
Per discussion, keep `filter` use unless `Q` required for complex queries